### PR TITLE
Render data

### PR DESCRIPTION
**Pull request by xueccc**
-query shop and products details from database
-replace hardcode info with data from database
-add styling to product images and names

### CODE REVIEW

**Comment by xueccc** [comment]
@ChuaJ Hi Jess, would you please review my PR?

File: client/src/components/productGrid.jsx [hunk 1]
```diff
@@ -4,20 +4,15 @@ import ProductInfo from './productInfo.jsx'
...
-      </div>
-      <div className="productGrid">
-      <ProductInfo />
-      <ProductInfo />
-    </div>
-    <div className="productGrid">
-      <ProductInfo />
-      <ProductInfo />
-    </div>
+      {Object.entries(props.shopProducts).map((item, key) => {
```
Comment (ChuaJ) [inline client/src/components/productGrid.jsx]: You list key as a parameter but do not use it in the return statement. You can take it out of the parameters!

File: client/src/components/productGrid.jsx [hunk 2]
```diff
@@ -4,20 +4,15 @@ import ProductInfo from './productInfo.jsx'
...
+            <ProductInfo prodInfo={item} />
+          </div>
+        );
+      })}
   </div>
   )
 }
 
-export default ProductGrid;
+export default ProductGrid;
```
Comment (ChuaJ) [inline client/src/components/productGrid.jsx]: New line!

File: client/src/index.jsx [hunk 3]
```diff
@@ -5,23 +5,65 @@ import 'bootstrap';
...
+        shop_image_url: null
+      },
+      shopProducts : {}
+    }
+    this.getShopProductInfo();
+  }
+
+  getShopProductInfo() {
+    axios.get(`${window.location.href}shopproducts`)
+    .then(response => {
```
Comment (ChuaJ) [inline client/src/index.jsx]: Your "then" method should be indented as well as the contents inside of its callback function

File: client/src/index.jsx [hunk 4]
```diff
@@ -5,23 +5,65 @@ import 'bootstrap';
...
+        id: item.id,
+        price: item.price,
+        liked: item.liked,
+        imgs_url: [item.image_url]
+      }
+    })
+    this.setState({
+      shopInfo: response.data[0][0],
+      shopProducts: sortByProductsId 
+    })
```
Comment (ChuaJ) [inline client/src/index.jsx]: Fixing indentation in this "then" method will greatly improve its readability!

File: client/src/index.jsx [hunk 5]
```diff
@@ -5,23 +5,65 @@ import 'bootstrap';
...
+    })
+    .catch(error => {
+      console.log('Error: ', error)
+    })
   }
+
+  componentDidMount() {
```
Comment (ChuaJ) [inline client/src/index.jsx]: Since you call your GET request function in the constructor you don't need this lifecycle method (for now).

**Comment by xueccc** [comment]
@ChuaJ Hi Jesse, I made updates based on your suggestion and implemented Airbnb linter. Would you please take a look?

**Pull request merged**